src/main/plugin/iso19139.ca.HNAP/loc/eng/schematron-rules-multilingual.xml

```diff
@@ -41,6 +41,8 @@
   <ResourceDescriptionFormat>Online resource description format is not valid. Valid values are:</ResourceDescriptionFormat>
   <ResourceDescriptionLanguage>Online resource description language is not valid. Should be a comma separated values of ISO-LANG-3 codes</ResourceDescriptionLanguage>
   <SecurityLevel>Security User Note is not valid. Valid values are:</SecurityLevel>
+  <SecurityLevelEn>Security User Note is not valid and required by both languages. Valid English values are: </SecurityLevelEn>
```

Comment:
Why not stay consistent with all other messages- i.e. OtherConstraintsNote2, Keyword, <OrganisationNameThesaurus
And simply put 
`Security User Note is required by both languages`

If you really need to include the options then I would recommend using placeholders.  Using multiple message keys to display a message is messy as different languages can change the order of the sentences causing all sorts of issues.  Also it is not clear by looking at the message the order they will need to be concatenated making hard for translators to get the full context of the messages without looking into the code to see how the message is created.